Index: aclocal.m4
===================================================================
RCS file: /export/cvsroot/profit/aclocal.m4,v
retrieving revision 1.7
diff -u -r1.7 aclocal.m4
--- aclocal.m4	2001/05/04 16:58:34	1.7
+++ aclocal.m4	2001/05/04 19:15:48
@@ -2072,7 +2072,7 @@
     old_archive_from_new_cmds='true'
     # FIXME: Should let the user specify the lib program.
     old_archive_cmds='lib /OUT:$oldlib$oldobjs$old_deplibs'
-    fix_srcfile_path='`cygpath -w "$srcfile"`'
+    fix_srcfile_path='`CYGWIN= cygpath -w "$srcfile"`'
     ;;
 
   darwin* | rhapsody*)
